acctz/acctz.proto

```diff
@@ -284,14 +284,24 @@ message GrpcService {
 
   // A string of bytes that are gRPC encoded. Best to decode using
   // PROTO_ANY.
-  repeated google.protobuf.Any payloads = 3;
+  // Deprecated in favour of using the payload field.
+  // A warning when using this field: if a google.protobuf.Any value
+  // is truncated, then it will not be possible for the client to
+  // unmarshal this value upon receiving.
+  repeated google.protobuf.Any payloads = 3 [deprecated=true];
 
-  // True, if truncation of payloads occurs due to an implementation
+  // True, if truncation of payload occurs due to an implementation
   // limitation in the originating service, any middleware, or the receiver.
   bool payload_istruncated = 5;
 
   // Authorization details
   AuthzDetail authz = 6;
+
+  // A string describing the payload of the RPC.
+  // This may be truncated.
+  // It also may be partially redacted to avoid leaking
+  // sensitive info (e.g. private keys rotated via certz/credentialz)
+  string payload = 7;
```

Comment:
in general i think the proto any is nice as it allows for parsing payload for rpc services or other services which can provide it 
Agree that logging this representation will truncate the bytes and cause an error - if you "stringified" the proto before logging though it would "mainly" readable

Reply:
i would see us changing this to be a oneof though to allow a string type for "simple logging" then a proto any for "structured logging"

Reply:
ack - made this be a oneof in the latest commit to allow either `string` or `google.protobuf.Any` to be sent